src/expr/src/vector_op/timestamptz.rs

```diff
@@ -62,6 +65,39 @@ pub fn timestamp_at_time_zone(input: NaiveDateTimeWrapper, time_zone: &str) -> R
     Ok(usec)
 }
 
+pub fn timestamptz_to_string(elem: i64, time_zone: &str, writer: &mut dyn Write) -> Result<()> {
+    let time_zone = lookup_time_zone(time_zone)?;
```

Comment:
So the `time_zone` string will be looked up for many times. Is there any way to optimize it? Because in most cases this will just a literal. (We can even reject the non-literal cases, I think)

Reply:
The lookup is a hash table lookup, with hashing cost linear to string length. The bigger cost is actually our way of evaluating literal expr, which would create a column full of duplicated values.
The ideal case, of course, is to pass a single session timezone as task context from frontend to compute node, and do a single lookup. But I guess we can defer that refactor until there is a more useful case for expr eval context.

Reply:
But it is not a fixed offset. The offset can vary at different dates.

Reply:
Hmm, indeed. We could represent it by the chrono_tz enum value, and repr in our type system as `i32`? 

https://docs.rs/chrono-tz/latest/chrono_tz/static.TZ_VARIANTS.html# 

It may be a bit brittle. But an i32 at the RHS of `AT TIME ZONE` will always be interpreted as a chrono_tz timezone.

Reply:
Btw, we don't support strings such as `UTC+8`. Do you think we should support this as well? We can reserve some space in our `i32` encoding for such strings.

Reply:
created issue: https://github.com/risingwavelabs/risingwave/issues/7335